### ISSUE TITLE
tensorflow vgg 16 can't inference after convert to xmodel

### ISSUE THREAD

**Issue by HuiJu1218** [Xilinx/Vitis-AI#1012]
Hi,

Here is my quantize.model and xmodel
[quantize model and xmodel](https://drive.google.com/drive/folders/1br27QEFIp94VolTkD4Rwg51NpmXBPVgn?usp=sharing)

my environment:
- vitis-ai 1.4 cpu
- convert model in tensorflow1 container

I had use keras to get vgg 16 pre-trained model and set model like following. 
![vgg](https://user-images.githubusercontent.com/55121392/190337351-0ce1da1f-80ff-4cb4-ad19-230d8a6702ef.png)

![vgg-output](https://user-images.githubusercontent.com/55121392/190337864-ca94155d-4d56-4b87-9a47-9f8496975e94.png)
And I had convert the vgg 16 model to xmodel successfully. But when I starting inference with xmodel, I get error message. The message seems like that model get error output layer.
![螢幕擷取畫面 2022-09-02 160255](https://user-images.githubusercontent.com/55121392/190339650-d3b40a24-3d2a-458c-8592-a6bc453670d8.png)
I have several question. Does anyone can help me?
1.  I have use netron to compare the xmodel from vitis-ai model zoo and the xmodel that I convert form tensorflow. It looks very different.
2.  The error message shows that correct output must be [1, 2], but get output [1, 2048].

![螢幕擷取畫面 2022-09-15 152905](https://user-images.githubusercontent.com/55121392/190349256-20bc45b7-221e-4939-8515-7c590584d936.png)

**Comment by lishixlnx** [comment]
open your .xmodel, you can see:
![image](https://user-images.githubusercontent.com/79431584/190560479-066d75f1-b2a5-4d58-9df3-bb84f8d57c3b.png)

there is a cpu-subgraph in the model, the [1, 2048] is the output of the "download" node point to the cpu subgraph.  the output of the 2nd Dpu-subgraph is [1,2],  which is what you really need.
The key problem is : the model is divided by 1 cpu-subgrpah, this is because there's op that dpu doesn't support. 
from the structure, it's mainly because of the "add" & "mul" op.

**Comment by Innodisk-Will** [comment]
The "add" & "mul" op is "Batchnormalization layer" convert to ".pb" get it. Then, I will remove the "Batchnormalization layer" and training, and convert it to ".pb" & ".xmodel" can run it in KV260. This might not be the best solution, but this works for me.
![xmodel](https://user-images.githubusercontent.com/87640491/191151403-2bf3f1cf-3ff5-432c-8ff8-675fc4911063.png)
